src/main/markers/waveform.segments.js

```diff
@@ -25,10 +25,10 @@ define([
       return view;
     });
 
+    var segmentId = 0;
     var createSegmentWaveform = function (options) {
-      var segmentId = "segment" + self.segments.length;
       var segment = {
-        id: segmentId,
+        id: options.id || ('peaks.segment.' + segmentId++),
```

Comment:
I've just updated waveform-data.js to v1.5.2, which fixes this issue. I can't publish a new version to npm as this needs @oncletom, who's currently away, so for now I've added a git URL to package.json.